src/torrent.cpp

```diff
@@ -2022,8 +2018,7 @@ namespace libtorrent
 
 		// in case file priorities were passed in via the add_torrent_params
 		// and also in the case of share mode, we need to update the priorities
-		if (!m_file_priority.empty() && std::find(m_file_priority.begin()
-				, m_file_priority.end(), 0) != m_file_priority.end())
+		if (std::find(m_file_priority.begin(), m_file_priority.end(), 0) != m_file_priority.end())
```

Comment:
I would prefer to keep lines shorter than this

Reply:
I'm not aware of current line length limitation so I assumed to align this to the longest line(# 2061) of the function.

Reply:
Creating actual '.clang-format' file may be a good idea.

Reply:
yes, I've considered that. I haven't had time to learn how to describe the existing formatting rules in libtorrent though. I didn't realise you had change this line other than reformatting it. This is also not a functional change, right?

Reply:
I'll try helping with this.

The reason why I changed this line, is that I thought `!m_file_priority.empty()` expression could be redundant here. `std::find` will work for an empty container just as well.

Reply:
If you don't like it, I'll revert this change easily.

Reply:
no, this is fine